### ISSUE TITLE
Products don't sync and there is no notification of failing.

### ISSUE THREAD

**Issue by k1rah** [bluevisiontec/GoogleShoppingApi#55]
When I try to sync products, nothing happens. I tried syncing 1-9 products and the spinning icon shows for a few seconds, then nothing happens. 

EDIT: It appears that the feed is case sensitive. For example, my attribute 'Size' causes sync to fail. The log says 'required attribute 'size' is missing.' It also does this for age group and gender when the value is capitalized (Adult, Female, etc..)

**Comment by ofnorms** [comment]
Attributes are mapped correctly but this error appears on every single product. Theoretically I could redo all my attribute values to lower case, but the values with numbers still fail.
![googleapifail](https://cloud.githubusercontent.com/assets/13491598/9241276/255e4f7a-4126-11e5-8ba7-e8aee40ea379.png)

**Comment by schnere** [comment]
You are trying to add simple products, not configurable?
There shouldn't be a problem with case sensitivity or numbers. Examples provided from Google:

XXS, XS, S, M, L, XL, 1XL, 2XL, 3XL, 4XL, 5XL, 6XL
00, 0, 02, 04, 06, 08, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34
Even "30 Short", "30 Regular", "30 Long" or "Small", "Medium", "Large" would be okay.

I will add "size_system" and "size_type" in a future commit, as this could be a possible reason to cause the error?

**Comment by k1rah** [comment]
My site has a mix of both, but I was just trying on simple products.

I did check with Google to make sure I had things correct.
I also double checked all attribute mapping. 
My products definitely have all required Google attributes filled out.

![screenshot from 2015-08-20 12 46 21](https://cloud.githubusercontent.com/assets/13753652/9393857/7de3da8e-4739-11e5-8c98-0da1626d8af4.png)

And this is the same product I showed in the above screenshot with errors:
![screenshot from 2015-08-20 12 51 59](https://cloud.githubusercontent.com/assets/13753652/9393980/5003b9bc-473a-11e5-8933-87bd77108716.png)
